### PR TITLE
Fix various rippled api parameters 

### PR DESCRIPTION
**Pull request by wilsonianb**
More fallout from https://github.com/ripple/ripple-lib/pull/843 😄

### CODE REVIEW

File: content/rippled-api-methods/account_offers.md [hunk 1]
```diff
@@ -163,8 +163,8 @@ The response follows the [standard format](#response-formatting), with a success
...
 | `ledger_index`         | Integer                                    | (Omitted if `ledger_current_index` provided instead) Sequence number, provided in the request, of the ledger version that was used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
-| `ledger_hash`          | String                                     | (May be omitted) Hex hash, provided in the request, of the ledger version that was used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
-| `marker`               | [(Not Specified)](#markers-and-pagination) | Server-defined value indicating the response is paginated. Pass this to the next call to resume where this call left off. Omitted when there are no pages of information after this one. [New in: rippled 0.26.4][] |
+| `ledger_hash`          | String                                     | _(May be omitted)_ Hex hash, provided in the request, of the ledger version that was used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
```
Comment (intelliot) [inline content/rippled-api-methods/account_offers.md:166]: Why is `(May be omitted)` italicized?

I see the phrase used a few other places ([example](https://github.com/wilsonianb/ripple-dev-portal/blob/c3124b6991617257b1c98c358715a590ec150eb2/content/rippled-api-methods/account_lines.md)) where it's not in italics.

**Review by intelliot — APPROVED** [review]
Added 1 question. Otherwise LGTM

File: content/rippled-api-methods/account_offers.md (hunk 1 above)
Comment (wilsonianb) [reply content/rippled-api-methods/account_offers.md:166]: The example(s) I looked at had it italicized, but it does look like more are not.
@mDuo13 Which side do you lean on?

Comment (mDuo13) [reply content/rippled-api-methods/account_offers.md:166]: Italicize all the parentheticals! Well, at least all the ones that start a Description column.

(Some of the older stuff pre-dates this rule. Feel free to update cases you see that aren't italicized.)

File: README.md [hunk 2]
```diff
@@ -9,16 +9,15 @@ Repository Layout
 
 The HTML pages in this portal are generated from the markdown files in the [content/](content/) folder. Always edit the markdown files, not the HTML files. The [assets/](assets/) folder contains static files used by the site's templates. The [img](img/) folder contains images used in the docs.
 
-The HTML files are generated using Ripple's documentation tool, called [**Dactyl**](https://github.com/ripple/dactyl). After you've done the [Dactyl Setup](#dactyl-setup), you can build the docs from the `tool/` folder:
+The HTML files are generated using Ripple's documentation tool, called [**Dactyl**](https://github.com/ripple/dactyl). After you've done the [Dactyl Setup](#dactyl-setup), you can build the docs from the project root directory:
```
Comment (mDuo13) [inline README.md:12]: Good catch. Been a while since I updated that file.

**Comment by mDuo13** [comment]
👍 and confirmed the build failures are unrelated to these edits.

**Pull request merged**